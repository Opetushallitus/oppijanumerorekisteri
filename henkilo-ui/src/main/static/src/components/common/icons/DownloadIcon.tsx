import React from 'react';

const DownloadIcon = () => <i className="fa fa-download" aria-hidden="true" style={{ paddingLeft: '1em' }} />;

export default DownloadIcon;
