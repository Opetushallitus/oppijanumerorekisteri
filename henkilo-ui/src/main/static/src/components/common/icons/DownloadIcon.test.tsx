import { shallow } from 'enzyme';
import DownloadIcon from './DownloadIcon';

describe('DownloadIcon', () => {
    test('Should render without crashing', () => {
        expect(shallow(<DownloadIcon />)).toMatchSnapshot();
    });
});
