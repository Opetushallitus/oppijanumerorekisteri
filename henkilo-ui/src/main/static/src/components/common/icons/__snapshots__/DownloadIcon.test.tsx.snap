// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DownloadIcon Should render without crashing 1`] = `
<i
  aria-hidden="true"
  className="fa fa-download"
  style={
    Object {
      "paddingLeft": "1em",
    }
  }
/>
`;
